### ISSUE TITLE
Expose the unblock code, totp and session email verification code sizes via API

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#12290]
The VPN client has these 3 values written in the stone:

* unblock code - length 8 digits
* totp code - length 6 digits
* session email verification code - length 6 digits

It would be nice to have an API endpoint to fetch these sizes so that, if FxA changes them, the client is always up-to-date. Thoughts?

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-4780)

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

Bonus goal: can we have the range of charts? In this way we can pick the right virtual keyboard on mobile. Only digits, alpha-numeric, etc..

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Thanks for filing this.  Your bonus goal sounds like a good idea too.  We can add that to our API documentation.

**Comment by cknowles-admin** [comment]
FxA is no longer synchronizing all issues between Jira and Github.  We are closing open issues and will selectively synchronize in the future.
